### PR TITLE
BOM-2050: Post Python 3.8

### PR DESCRIPTION
**Pull request by mzulqarnain1**
Relevant JIRA : https://openedx.atlassian.net/browse/BOM-2050

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,6 +8,7 @@ services:
 
 env:
   - TOXENV=django22
+  - TOXENV=django30
```
Comment (mraarif) [inline .travis.yml:11]: shouldn't we also add a worker for Django 3.1

**Review by UsamaSadiq — APPROVED** [review]

File: openedxstats/settings/base.py [hunk 2]
```diff
@@ -115,7 +115,7 @@
 #)
 
 # Whitenoise compression config
-STATICFILES_STORAGE = 'whitenoise.django.GzipManifestStaticFilesStorage'
+STATICFILES_STORAGE = 'whitenoise.storage.CompressedManifestStaticFilesStorage'
```
Comment (mraarif) [inline openedxstats/settings/base.py:118]: have we verified that this works as same as GZip

Comment (mzulqarnain1) [reply openedxstats/settings/base.py:118]: http://whitenoise.evans.io/en/stable/changelog.html#v4-0

File: .travis.yml (hunk 1 above)
Comment (mzulqarnain1) [reply .travis.yml:11]: Sure, I'll add that one too.

**Review by mraarif — APPROVED** [review]

**Review by jmbowman — APPROVED** [review]

**Comment by jmbowman** [comment]
@nedbat Heads up on this in case you want to try deploying it sometime after the Koa release chaos dies down.  Not urgent at all, just might be nice to make sure it deploys cleanly before doing further Django upgrade work, etc.

**Pull request merged**